Formula/glab.rb

```diff
@@ -16,6 +16,8 @@ class Glab < Formula
   depends_on "go" => :build
 
   def install
+    ENV["CGO_ENABLED"] = "1"
```

Comment:
Do we want this set on Linux too, or is this only useful on macOS? In the latter case, we can do
```suggestion
    on_macos { ENV["CGO_ENABLED"] = "1" }
```
Can we test that this change does what we want it to? (i.e. use the native DNS resolver on macOS)

Reply:
Checking for macOS is a very good suggestion.

Regarding the test: I consider this quite hard to test as the test would only work on macOS und require quite a complex setup. But I actually do not think that testing this is actually necessary as using the native resolver when compiling with CGO is a documented "feature" in Golang, so no hack or workaround. ;)

Reply:
I added your suggestion in the amended commit, I hope this is fine ;)